libs/community/langchain_community/retrievers/you.py

```diff
@@ -1,13 +1,16 @@
 from typing import Any, List
 
-from langchain_core.callbacks import CallbackManagerForRetrieverRun
+from langchain_core.callbacks import (
+    AsyncCallbackManagerForRetrieverRun,
+    CallbackManagerForRetrieverRun,
+)
 from langchain_core.documents import Document
 from langchain_core.retrievers import BaseRetriever
 
 from langchain_community.utilities import YouSearchAPIWrapper
 
 
-class YouRetriever(BaseRetriever, YouSearchAPIWrapper):
+class YouRetriever(BaseRetriever, YouSearchAPIWrapper):  # type: ignore[valid-type,misc]
```

Comment:
why the type hint to ignore?

Reply:
@hwchase17 development artifact. Removed

thanks